mtags/src/main/scala-3/scala/meta/internal/pc/PcDocumentHighlightProvider.scala

```diff
@@ -262,7 +274,8 @@ object PcDocumentHighlightProvider:
 
         val traverser = new DeepFolder[Set[DocumentHighlight]](collectNames)
         val all = traverser(Set.empty[DocumentHighlight], unit.tpdTree)
-        all.toList.distinct
+
+        all.toList.distinctBy(_.getRange())
```

Comment:
Looks like Scala 3 has several trees corresponding to the same position, but this workaround seems to work ok.